### PR TITLE
Mayland Blocks: Added post meta

### LINKED ISSUE #3551: Mayland Blocks: Add entry meta to posts
Bring the change from this PR to Mayland Blocks: https://github.com/Automattic/themes/pull/3541/commits/ce552f25374fd4ecefe2eae67c5d4505f0ed479d since the design is almost the same

### PR DESCRIPTION
**Pull request by MaggieCabrera**
#### Changes proposed in this Pull Request:

This PR adds the post meta template part and adjusts it to fit Mayland's design and icons.

#### Related issue(s):

Closes #3551

### CODE REVIEW

**Comment by MaggieCabrera** [comment]
TODO:

- [x] Change the color of the links to be gray
- [x] Add the post tag block too
- [x] Index template also needs the post meta template part

File: mayland-blocks/assets/svg/post-date.svg [hunk 1]
```diff
@@ -1,9 +1,9 @@
 <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" xmlns:xlink="http://www.w3.org/1999/xlink">
     <defs>
-        <path id="a" d="M0 0h24v24H0V0z"></path>
+        <path fill="#666666" fill-opacity="1" id="a" d="M0 0h24v24H0V0z"></path>
```
Comment (MaggieCabrera) [inline mayland-blocks/assets/svg/post-date.svg:3]: Sadly there's no better way of doing this. Loading the svg's as background images makes it impossible for us to have dynamic fill colors. Hopefully something like an [Icon block](https://github.com/WordPress/gutenberg/issues/16484) will be a better solution in cases like this

File: mayland-blocks/block-templates/index.html [hunk 2]
```diff
@@ -9,7 +9,7 @@
 
 <!-- wp:post-excerpt /-->
 
-<!-- wp:post-date /-->
+<!-- wp:template-part {"slug":"post-meta","theme":"a8c-themes/mayland-blocks"} /-->
```
Comment (MaggieCabrera) [inline mayland-blocks/block-templates/index.html]: The template part doesn't seem to be working unless we add the theme slug, which is very inconvenient

**Review by scruffian — APPROVED** [review]
LGTM

**Pull request merged**